extensions/2.0/Vendor/GODOT_single_root/GODOT_single_root.md

```diff
@@ -0,0 +1,83 @@
+# GODOT_single_root
+
+## Contributors
+
+* Aaron Franke, The Mirror Megaverse Inc.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension imposes constraints on glTF scenes to ensure they have a single root node.
+
+Many games engines import glTF scenes as objects with a single root node. Unity has prefabs with a single root GameObject, and Godot has scenes with a single root node. Since glTF allows defining multiple root nodes, engines will insert the glTF root nodes as children of the "real" root node, which makes it difficult for a glTF file to define the properties of the "real" root node. This is important for things like physics bodies and character controllers, which are defined on the root node, so that all child nodes are moved with the body. Aside from physics, a single root node would avoid an extra node in the tree when round-tripping between glTF and these engines.
+
+The `GODOT_single_root` glTF extension solves this problem by imposing additional constraints on the glTF scene to ensure it can be parsed into one of these objects with a single root node. Implementations can detect `GODOT_single_root` and import the single root node as the object's "real" root node in the scene/prefab/etc. The extension contains no data, it only restricts behavior. The restrictions are as follows:
+
+* The document `"scenes"` array MUST have exactly one scene.
+* The document `"scene"` index MUST be set to 0, the index of the first scene in the `"scenes"` array.
```

Comment:
These restrictions contradict the language above:
> import glTF scenes as objects with a single root node

and

> constraints on the glTF scene to ensure it can be parsed into one of these objects with a single root node

Either the overview needs to be adjusted to provide rationale for disallowing multiple scenes, which is orthogonal to root nodes, or these two restrictions should be lifted.